packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -760,9 +762,13 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
                     selection: TextSelection.collapsed(offset: entry.label.length),
                   );
                   widget.onSelected?.call(entry.value);
+                      _localTextEditingController?.dispose();
```

Comment:
The same comment here.

It will cause recreate of controller on each onEditingComplete.